static/js/public/snap-details/channelMap.js

```diff
@@ -177,12 +175,13 @@ class ChannelMap {
     this.channelOverlayEl.style.display = 'block';
     this.channelMapEl.classList.remove('is-closed');
 
-    if (typeof ga !== 'undefined') {
+    // As discussed with David Calle, we should just track 'install' button clicks
+    if (this.openScreenName === 'channel-map-install' && typeof ga !== 'undefined') {
```

Comment:
If it's meant to be tracking clicking on 'Install' button I think a better place for this would be a click event handler, not a function that renders channel map dialog.

Reply:
Moved to the click handler, thanks!